utils/simulations_v1_water_cycle.csv

```diff
@@ -0,0 +1,52 @@
+v1, WaterCycle, LR, DECK, 20180302.DECKv1b_H3.ne30_oEC.edison, edison, , historical, 3, , ,
+v1, WaterCycle, LR, DECK, 20180305.DECKv1b_H4.ne30_oEC.edison, edison, , historical, 4, , ,
+v1, WaterCycle, LR, DECK, 20180307.DECKv1b_H5.ne30_oEC.edison, edison, , historical, 5, , ,
+v1, WaterCycle, LR, DECK, 20180316.DECKv1b_A1.ne30_oEC.edison, edison, , amip, 1, , ,
+v1, WaterCycle, LR, DECK, 20180622.DECKv1b_A2.ne30_oEC.edison, edison, , amip, 2, , ,
+v1, WaterCycle, LR, DECK, 20180716.DECKv1b_A3.ne30_oEC.edison, edison, , amip, 3, , ,
+v1, WaterCycle, LR, DECK, 20180508.DECKv1b_A1_1850allF.ne30_oEC.edison, edison, , amip_1850allF, 1, , ,
+v1, WaterCycle, LR, DECK, 20180622.DECKv1b_A2_1850allF.ne30_oEC.edison, edison, , amip_1850allF, 2, , ,
+v1, WaterCycle, LR, DECK, 20180716.DECKv1b_A3_1850allF.ne30_oEC.edison, edison, , amip_1850allF, 3, , ,
+v1, WaterCycle, LR, DECK, 20180508.DECKv1b_A1_1850aeroF.ne30_oEC.edison, edison, , amip_1850aeroF, 1, , ,
+v1, WaterCycle, LR, DECK, 20180622.DECKv1b_A2_1850aeroF.ne30_oEC.edison, edison, , amip_1850aeroF, 2, , ,
+v1, WaterCycle, LR, DECK, 20180716.DECKv1b_A3_1850aeroF.ne30_oEC.edison, edison, , amip_1850aeroF, 3, , ,
+v1, WaterCycle, LR, DECK, 20190404.DECKv1b_H1_hist-GHG.ne30_oEC.edison, edison, , damip_hist-GHG, 1, , ,
+v1, WaterCycle, LR, DECK, 20190404.DECKv1b_H2_hist-GHG.ne30_oEC.edison, edison, , damip_hist-GHG, 2, , ,
+v1, WaterCycle, LR, DECK, 20190404.DECKv1b_H3_hist-GHG.ne30_oEC.edison, edison, , damip_hist-GHG, 3, , ,
+v1, WaterCycle, LR, DECK, 20190807.DECKv1b_P1_SSP5-8.5.ne30_oEC.cori-knl, cori-knl, , projection, 1, , ,
+v1, WaterCycle, LR, DECK, 20190821.DECKv1b_P2_SSP5-8.5.ne30_oEC.cori-knl, cori-knl, , projection, 2, , ,
+v1, WaterCycle, LR, DECK, 20190821.DECKv1b_P3_SSP5-8.5.ne30_oEC.cori-knl, cori-knl, , projection, 3, , ,
+v1, WaterCycle, LR, DECK, 20190821.DECKv1b_P4_SSP5-8.5.ne30_oEC.cori-knl, cori-knl, , projection, 4, , ,
+v1, WaterCycle, LR, DECK, 20190821.DECKv1b_P5_SSP5-8.5.ne30_oEC.cori-knl, cori-knl, , projection, 5, , ,
+v1, WaterCycle, LR, DECK, 20191019.DECKv1b_P1_SSP5-8.5-GHG.ne30_oEC.cori-knl, cori-knl, , damip_ssp5-8.5-GHG, 1, , ,
+v1, WaterCycle, LR, DECK, 20191019.DECKv1b_P2_SSP5-8.5-GHG.ne30_oEC.cori-knl, cori-knl, , damip_ssp5-8.5-GHG, 2, , ,
+v1, WaterCycle, LR, DECK, 20191019.DECKv1b_P3_SSP5-8.5-GHG.ne30_oEC.cori-knl, cori-knl, , damip_ssp5-8.5-GHG, 3, , ,
+v1, WaterCycle, HR, DECK, 20211021-maint-1.0-tro.A_WCYCLSSP585_CMIP6_HR.ne120_oRRS18v3_ICG.unc12-3rd-attempt, ???, , ssp5_8.5, 1, , ,
+v1, WaterCycle, HR, DECK, 20200517-maint-1.0-tro.A_WCYCL20TRS_CMIP6_HR.ne120_oRRS18v3_ICG.unc11, ???, , , , , ,
+v1, WaterCycle, HR, DECK, 202101027-maint-1.0-tro.A_WCYCL20TRS_CMIP6_HR.ne120_oRRS18v3_ICG.unc12, ???, , , , , ,
+v1, WaterCycle, HR, DECK, theta.20190910.branch_noCNT.n825def.unc06.A_WCYCL1950S_CMIP6_HR.ne120_oRRS18v3_ICG, , theta, , , , , 
+v1, WaterCycle, HR, DECK, 20210112.A_WCYCL1950S_CMIP6_HR.ne120_oRRS18v3_ICG.unc06, ???, , , , , ,
```

Comment:
For 4 simulations, I couldn't deduce what machine they were run on.

Reply:
Don't know either. Maybe that's not important.